server/api/controller/plan.js

```diff
@@ -63,135 +64,146 @@ class PlanController extends Controller {
 			// multiplied to get an approximate meters value
 			const spatialUnitFactor = Config.locationSearch.dbDistanceInMeters ? 1 : 111195;
 
-			q.columns.push(Knex.raw(`ST_Distance(geom, ST_GeomFromText("${polygon}",4326))*${spatialUnitFactor} as distance`));
+			//   console.log(geom);
+			q.columns.push(
+				Knex.raw(
+					`ST_Distance(ST_GeomFromText(ST_AsText(geom),4326), ST_GeomFromText("${polygon}",4326))*${spatialUnitFactor} as distance`
```

Comment:
@harshitsan  why did you change `geom` to `ST_GeomFromText(ST_AsText(geom)`?

Reply:
Yes, Actually this is not required, This can be reverted to the master branch.